core/src/main/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
```

Comment:
main sources shouldn't have logback configuration - that should be left to the apps

Reply:
fixed